scss/mixins/_text-hide.scss

```diff
@@ -1,8 +1,14 @@
 // CSS image replacement
 @mixin text-hide() {
   font: "0/0" a;
-  color: transparent;
   text-shadow: none;
   background-color: transparent;
   border: 0;
+
+  &,
+  &:hover,
```

Comment:
These should be using `::hover` as we no longer support IE8 (IIRC the old single-colon syntax is deprecated in the spec as it does not work well with xml namespaces).